### PR TITLE
adds sistr 1.1.2

### LINKED ISSUE #1001: [Request An Update]: sistr
### What container needs an update?

There's a new version of sistr, but the version 1.1.1 Dockerfile doesn't have any tests and needs to be updated.

### PR DESCRIPTION
**Pull request by Kincekara**
Closes #1001 
I dare to update this tool and I regret it. 
pip installation fails because of legacy libraries. mamba image became too big. So, I ended up using setup.py. 
I added a readme file and a test. It looks like it is working but it may be good to test it further.

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: sistr/1.1.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+WORKDIR /data
+
+ENV LC_ALL=C
+
+CMD ["sistr", "--help"]
+
+## Test ##
+FROM app AS test
+
+RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/006/945/GCF_000006945.2_ASM694v2/GCF_000006945.2_ASM694v2_genomic.fna.gz &&\
```
Comment (kapsakcj) [inline sistr/1.1.2/Dockerfile:46]: could you please combine the RUN layers on 46 and 49?

Also would be helpful to provide a comment line that mentions this is the famous LT2 strain and that we expect to see the Typhimurium serotype predicted 

and maybe another comment line with a link here: https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000006945.2/

File: sistr/1.1.2/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Main tool: [sistr](https://github.com/phac-nml/sistr_cmd)
+  
+Code repository: https://github.com/phac-nml/sistr_cmd
+
+Additional tools:
+- blast : 2.12.0+ 
+- mafft : v7.490
+- mash : 2.3
+
```
Comment (kapsakcj) [inline sistr/1.1.2/README.md:11]: could you please add pandas v1.3.5 and numpy 1.21.5? They seem to be important python packages

**Comment by kapsakcj** [comment]
>_I dare to update this tool and I regret it.
pip installation fails because of legacy libraries. mamba image became too big. So, I ended up using setup.py._

We thank you for your efforts 👏 


It looks like there are some built-in tests that test a few different components. Could you try adding these test commands somewhere in the test stage and see what happens? see here: https://github.com/phac-nml/sistr_cmd?tab=readme-ov-file#development

**Comment by kapsakcj** [comment]
Darn OK. Thanks for trying anyways. Glad you added the additional test and thanks for making my requested changes.

I'm fine if you want to wait for v1.1.3, whatever you think is best. I don't think the changes between 1.1.2 and 1.1.3 look too drastic, but may be important for anyone working on serovars that are going to be updated in 1.1.3: https://github.com/phac-nml/sistr_cmd/blob/v1.1.3/CHANGELOG.md

**Comment by Kincekara** [comment]
There is something sketchy in the results. I am leaving this as a draft for now. I will do some more tests when I have time. In the meantime, if it is updated I'll change the version too.

**Comment by kapsakcj** [comment]
FYI sistr 1.1.3 was just released https://github.com/phac-nml/sistr_cmd/releases/tag/v1.1.3

**Comment by Kincekara** [comment]
:sob: :sob: :sob:

**Review by erinyoung — APPROVED** [review]

**Comment by erinyoung** [comment]
I just got to this PR today. I think it looks good to merge and deploy and 1.1.3 can come later.

**Comment by Kincekara** [comment]
Agreed! I am merging this one while it is still working. I will open an issue for 1.1.3

**Pull request merged**